src/main/java/com/panasi/interview_questions/repository/dto/QuestionDto.java

```diff
@@ -9,7 +9,8 @@ public class QuestionDto {
 	
 	private Integer id;
 	private String name;
+	private Integer categoryId;
 	private List<AnswerDto> answers;
-	private CategoryDto category;
+	private String author;
```

Comment:
and better to add author id, because, it can be easier to work with this data

Reply:
Added author id